src/components/LogView/FullLogLine.js

```diff
@@ -86,6 +98,23 @@ export default class FullLogLine extends React.Component<Props, State> {
         onMouseUp={this.handleMouseUp}
         onMouseDown={this.handleMouseDown}
       >
+        <FontAwesomeIcon
+          onClick={() => {
```

Comment:
What if instead of changing the hash in the current view (because presumably the user doesn't care about the initial scroll in their current workflow since they're already here), this instead copied the full URL (with next hash) to the clipboard?

Reply:
i actually like pushing the changes to the URL because the user can leverage bookmarking, history, autocomplete and other url features while they're building their shareable page. 
changing the hash in the current view is helpful because it represents which line number in the bookmark panel to highlight and we only jump to shareLine on initial load. 

I think if we were copy the next hash to clipboard, we will need to indicate to the user that their clipboard contents will be overwritten before (or after?) clicking the shareLine icon.

Reply:
I think we should leave the UX in the original form. @mvankeulen94 mentioned that users requested bookmarks and sharelines as separate features. The clipboard collision could disrupt some devs workflows even though it is "required" to copy the url at some point to share the line if didn't implement the "copy to clipboard" feature.

Reply:
I think it's fine to show a toast if you really want, but I'd expect that users will know copying anything will overwrite their current clipboard contents. Having a copy shareline button would still be consistent with the bookmark and shareline features being separate

Reply:
In general based on my conversations with users, they'd like a feature which is as similar to the HTML viewer's linking to lines functionality as possible, regardless of whether they use bookmarks or not. Having a visual indicator of line selection and being able to share that via a hyperlink were two critical requests for this. Bookmarks in general add complications to line share functionality, so I believe we should keep these as two separate concepts.

Reply:
Just to be clear, users specifically wanted a persistent visual indicator of the line copied to the clipboard? The UX I proposed satisfies the requirements that you mentioned without adding to the visual complexity with an additional column

Reply:
Yes, users would like the persistent visual indicator. I think the risk with removing the arrow is that now the visual indicator separating the line from other lines, including bookmarks, disappears. If a user sends a page with several bookmarks in view, and the selected line is at the top, they'd need to be aware of which line was intended to be sent, and keep track of that as they scroll around the file. Additionally, shifting from an interface of clicking a line and copy-pasting the URL to click-to-copy seems like it could be a source of friction, given the existing precedent for the former.